### ISSUE TITLE
Difference between audiobook, media overlays and tts

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1514]
I see it was requested in the a11y task force call on 2021-02-11 that I add a note to the optimized publication section explaining the differences between these concepts, but such a discussion doesn't appear germane to the section. Audiobooks aren't in scope of the specification, TTS is not an optimization, and full audio only falls into the category when it's not included with full text, as already explained.

I'd suggest putting a general explanation in the techniques document section on primary access modes instead. I think this discussion is helpful in terms of explaining when a primary mode of auditory should be specified (even if audiobooks don't fit the discussion, they could be mentioned in comparison).

The other options I can think of include:

- adding a description of the difference between media overlays and a pure audiobook to the media overlays section of the core specification (not sure TTS fits here)
- writing a separate informative note that explains the differences between the three to avoid shoehorning this into sections where it doesn't fully fit

We might even want to consider a mix of these (e.g. descriptions in the techniques and media overlays).

**Comment by GeorgeKerscher** [comment]
I like the idea of a separate note on this issue, because as a standalone note, it might reach a broader audience. It could be used in a range of publications that help people understand the differences and usages.

**Comment by avneeshsingh** [comment]
This is a generic information, not too specific to EPUB 3. Therefore one option can be to develop this stand alone note in some other publishing group like Publishing CG or Publishing BG and then provide links to this document from EPUB Accessibility, Audio Books and other specifications in which we need to clarify this.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-02-25-epub-a11y#section2) on 2021-02-25

List of resolutions:

- ***[Resolution No. 2](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-02-25-epub-a11y#resolution2):  delegate document for difference between Media Overlays, Audio books, read aloud etc to Publishing CG***

<details><summary><i>View the transcript</i></summary>

### 2. Explainer for difference between audio books, Media Overlays, and read aloud 

_See github issue [#1514](https://github.com/w3c/epub-specs/issues/1514)._




**Avneesh Singh:** last meeting we said we wanted to clarify this somewhere, and we were going to leave this to mgarrish  

**Matt Garrish:** i looked at it, and wasn't sure how this fits in  
… if this is just a general primer, i suggest an informative note  
… maybe something for the CG to come up with that could be referenced in various specs  
… otherwise, we should clarify the purpose behind this, and then include a more narrow discussion in the immediately relevant portion of the spec (e.g. maybe MO)  

**George Kerscher:** i see confusion about this from higher learning people all the time  
… i think we should hand this off to CG to write an explainer  
… its useful for training people, but it might not belong in a spec  

**Avneesh Singh:** and we could link to the explainer from the spec  

**Tzviya Siegman:** if the goal is to ask the documentation group in CG to work on this, they are pretty busy right now, it could take a while  
… i think this could be a very short document  

**Matt Garrish:** i just don't know that this is a WG note, we could still handle this when we're doing CG work  

**Avneesh Singh:** this isn't really related to specs, this is more a publishing issue  

**Tzviya Siegman:** Laura Brady had mentioned that public funding might be put towards writing a11y documentation  
… this might fall into that category  

> *George Kerscher:* +1

> *Tzviya Siegman:* +1

> **Proposed resolution: delegate document for difference between Media Overlays, Audio books, read aloud etc to Publishing CG** *(Avneesh Singh)*

> *Gregorio Pellegrino:* +1

> *Bill Kasdorf:* +1

> *Charles LaPierre:* +1

> *Matt Garrish:* +1

> *Matthew Chan:* +1

> *Juliette McShane:* +1

> *Ben Schroeter:* 0

> *Will:* +1

> *Avneesh Singh:* And we will link to it from EPUB accessibility

> *Gregorio Pellegrino:* may we have public funding for buying us coffee? :)

> ***Resolution #2: delegate document for difference between Media Overlays, Audio books, read aloud etc to Publishing CG***

</details>
